### DISCUSSION TITLE
List of supported models

### DISCUSSION THREAD

**Discussion by DannyBen** [tangramxyz/tangram#21] [Q&A] [answered]
Is there anywhere a list of models that are known to tangram?

Like, when the Overview page says "The best model is a **Gradient Boosted Tree Multiclass Classifier.**" - what are the other ones that the system can suggest?

**Comment by nitsky** [comment, marked as answer]
Hi @DannyBen, in the left sidebar, under "Training", click "Grid". There you can see a list of models that tangram tried before choosing the best one. Right now tangram trains linear and gradient boosted decision tree models with a range of hyperparameter settings.

If you want to customize the specific models trained with a configuration file, check out the docs [here](https://www.tangram.xyz/docs/guides/train_with_custom_configuration).

**Reply by DannyBen** [reply]
Thank you. I saw that list, just thought it was only the list of most appropriate models for my dataset.
